tests/integration/test_native_tracking.py

```diff
@@ -16,6 +16,9 @@
 TEST_MULTITHREADED_EXTENSION = HERE / "multithreaded_extension"
 TEST_NATIVE_EXTENSION = HERE / "native_extension"
 
+if "linux" not in sys.platform:
+    pytest.skip("tests only works on Linux", allow_module_level=True)
```

Comment:
nit: The message should either be "tests that only work on Linux" or "tests only working on Linux"